src/test/java/testresults/PostTestResultsNegTestTypesCancelledLvl.java

```diff
@@ -108,7 +110,7 @@ public void testResultsNullCertificateLink() {
         testResultsSteps.validatePostErrorData("certificateLink", "is not allowed");
     }
 
-
+    @Ignore("blocked by CVSB-9639")
```

Comment:
defect was fixed, please remove the @Ignore annotation

Reply:
Removed.